scripts/build_protodocs_docker.sh

```diff
@@ -6,8 +6,8 @@ set -o errexit -o nounset -o pipefail
 # so it doesn't need the pre-clean step
 
 user=$(id -u):$(id -g)
-protoc="docker run --rm --user ${user} --mount type=bind,source=$(pwd),target=/work --tmpfs /tmp:exec iov1/prototool-docker protoc"
-prototool="docker run --rm --user ${user} --mount type=bind,source=$(pwd),target=/work --tmpfs /tmp:exec iov1/prototool-docker prototool"
+protoc="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.0 protoc"
```

Comment:
Thank you.
I had this built local earlier... no wonder tests didn't fail for me here